tools/meta_proteome_analyzer/meta_proteome_analyzer.xml

```diff
@@ -4,7 +4,6 @@
     </description>
     <requirements>
         <requirement type="package" version="1.4.1">mpa-portable</requirement>
-        <requirement type="package" version="1">fonts-conda-ecosystem</requirement>
```

Comment:
Unfortunately this is needed (openjdk 11 only brings fontconfig). Can you restore it.